### ISSUE TITLE
Error with filtering on enums

### ISSUE THREAD

**Issue by iherger** [graphql-nexus/nexus-plugin-prisma#403]
Filtering on enums doesn't seem to be possible.

The following photon query works:
```ts
  const filteredPosts = await photon.posts.findMany({
    where: { status: "DRAFT" }
  });
```
whereas the corresponding GraphQL query (exposed through nexus/nexus-prisma) fails:
```graphql
{
  posts (where: {status: "DRAFT"}) {
    title
  }
}
```
with the error message
```
Expected type QueryFindManyPostFilter, found "DRAFT".
```
A reproduction repo is available here: https://github.com/iherger/prisma2-filterenums

After installing it, run `yarn seed` .

Run `yarn test` to check that the photon query works.

Run `yarn start` and then try the graphql query above for the error message.

**Comment by Weakky** [comment]
While this looks like a different issue, this is the same underlying problem than #331

**Comment by Weakky** [comment]
This was fixed by #408.

Make sure to update nexus-prisma on your reproduction for it to work. The reproduction still uses an old version of nexus-prisma. See https://github.com/prisma-labs/nexus-prisma/releases/tag/v0.4.1 to know how to update.

Closing now
